tests/test_core.py

```diff
@@ -77,33 +88,58 @@ def test_unique():
     assert actual == expected
 
 
-def test_load_esmcol_remote(pangeo_cmip6_col):
-    assert isinstance(pangeo_cmip6_col.df, pd.DataFrame)
+def test_load_esmcol_remote(zarr_aws_cesmle_col):
+    assert isinstance(zarr_aws_cesmle_col.df, pd.DataFrame)
+
+
+def test_getitem():
+    key = 'CMIP.CNRM-CERFACS.CNRM-CM6-1.historical.Amon.gr'
+    col = intake.open_esm_datastore(cdf_col_sample_cmip6)
+    x = col[key]
+    assert isinstance(x, intake.catalog.local.LocalCatalogEntry)
+
```

Comment:
We could also use a few tests using these new catalog entries. We want to make sure we can pass various combinations of arguments to the open functions and the combined datasets are as expected.

Reply:
It would also be good to test the construction of an Entry outside the catalog.

Reply:
👍

I added a few more tests